validator/accounts/accounts_exit.go

```diff
@@ -183,13 +184,22 @@ func prepareClients(cliCtx *cli.Context) (*ethpb.BeaconNodeValidatorClient, *eth
 	if dialOpts == nil {
 		return nil, nil, errors.New("failed to construct dial options")
 	}
+	for _, hdr := range strings.Split(cliCtx.String(flags.GrpcHeadersFlag.Name), ",") {
```

Comment:
Thanks for this. To accept and merge this PR, we are going to need a regression test. Do you mind adding one for this?

Reply:
Sure thing 👍

Reply:
@djoyahoy has added the test and the validator tests pass:

```
Target //validator:go_default_test up-to-date:
  bazel-bin/validator/go_default_test_/go_default_test
INFO: Elapsed time: 8.898s, Critical Path: 8.30s
INFO: 46 processes: 6 internal, 40 darwin-sandbox.
INFO: Build completed successfully, 46 total actions
//validator:go_default_test                                              PASSED in 1.2s
  WARNING: //validator:go_default_test: Test execution time (1.2s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
INFO: Build completed successfully, 46 total actions
```

We are however, seeing a failure in the end to end tests that I can't pin down:
```
> bazel test //endtoend:go_default_test --define=ssz=minimal
...
=== CONT  TestEndToEnd_MinimalConfig
    helpers.go:126: Ending time: 2021-01-26 16:55:28.701779 +0000 UTC m=+472.829059328
--- FAIL: TestEndToEnd_MinimalConfig (472.79s)
...
    --- FAIL: TestEndToEnd_MinimalConfig/validators_participating_epoch_5 (0.00s)
...
FAIL
================================================================================
Target //endtoend:go_default_test up-to-date:
  bazel-bin/endtoend/go_default_test_/go_default_test
INFO: Elapsed time: 493.339s, Critical Path: 487.68s
INFO: 65 processes: 4 internal, 61 darwin-sandbox.
INFO: Build completed, 1 test FAILED, 65 total actions
//endtoend:go_default_test                                               FAILED in 1 out of 2 in 475.8s
  Stats over 2 runs: max = 475.8s, min = 295.1s, avg = 385.4s, dev = 90.4s
  /private/var/tmp/_bazel_tusharshah/3f45103db17b7158700e6ae8b56c98c1/execroot/prysm/bazel-out/darwin-fastbuild/testlogs/endtoend/go_default_test/shard_1_of_2/test.log

INFO: Build completed, 1 test FAILED, 65 total actions
```